src/puppetlabs/trapperkeeper/config.clj

```diff
@@ -59,6 +59,12 @@
     #{".yaml" ".yml"}
     (yaml/parse-string (slurp file))))
 
+(defn override-restart-file-from-cli-data
+  [config-data cli-data]
+  (if-let [cli-restart-file (:restart-file cli-data)]
+    (assoc-in config-data [:global :restart-file] cli-restart-file)
```

Comment:
Yeah, a warning might be good here.  I guess even if logback isn't initialized by this point - which I don't know yet - we could still capture the warning somewhere - like to the process' stdout.  So maybe good regardless?